planet/clients/orders.py

```diff
@@ -344,27 +347,37 @@ async def poll(
         order_id: str,
         state: str = None,
         wait: int = 10,
-        verbose: bool = False
```

Comment:
On line 212 in `scripts/cli.py` in the function `download()`, the keyword argument "`verbose`" is given as: `cl.poll(str(order_id), verbose=True)`.

I see that `verbose` was purposefully removed and replaced with `report`, however the keyword `verbose` was not removed in the CLI script. When this keyword is removed as argument (i.e., `cl.poll(str(order_id))`), the CLI seems to act as expected.

Reply:
What a great catch! Thank you for being this thorough. I wish we had something in place to do integration tests involving the CLI. Ideally, they would have caught that. I have made a ticket for considering CLI integration tests as well (#327).

Reply:
This was actually fairly in-depth. I had to add in the ability to disable the StatusBar reporter, to honor the `quiet` flag. Fixed now!